hw20/roles/hw23/tasks/kernel_list.yml

```diff
@@ -0,0 +1,20 @@
+---
+#Print list of all installed kernels Ubuntu
```

Comment:
You can use only one block for both OS types. Just try to provide shell command as a variable.

Reply:
Optional you can copy script(using template) and use it to collect and print kernel versions.
So instead of 2 block you can have 2 tasks

Reply:
`cat` part can be included to "collect" part. So 2 blocks will be replaces with only one task.